remote-config.jsonc

```diff
@@ -33,7 +33,7 @@
           "message": "Join the DDEV Discord to support others and get support: https://discord.gg/hCZFfAMc5k"
         },
         {
-          "message": "Sign up for the weekly contribution training on Wednesdays, it's coming up soon! https://ddev.com/blog/contributor-training/"
+          "message": "Sign up for Contributor training, upcoming sessions at https://www.meetup.com/ddev-events/events/"
```

Comment:
Would it make sense to use a period instead of a comma and make it two sentences?

Reply:
Done